CONTRIBUTING.md

```diff
@@ -29,4 +29,3 @@ Lastly, run the test command from the console:
 - `git tag -a vX.X.X -m 'vX.X.X'`
 - `git push --tags`
 - `npm publish`
-- Deploy packaged plugin in [mapbox-gl-js/plugins](https://github.com/mapbox/mapbox-gl-js/tree/mb-pages/plugins)
```

Comment:
Should we include something here about the process for making sure the gl-js plugin docs are up to date with the most recent version, or are you planning on pulling that in automatically somehow on the gl-js side?